### ISSUE TITLE
Question on performance

### ISSUE THREAD

**Issue by STARK1234** [damo-cv/TransReID#30]
Hi, thanks for your awesome work.
I met some problem when trying to reproduce the results of vit-base on MSMT17. all default settings were not changed. I only got 58.2% mAP and 78.8% Rank1 that results is far lower than your reported, 61% mAP and 81% Rank1.

In addition, could you provide the config file that is used for ResNet50 to achieve 51.3% mAP?

**Comment by michuanhaohao** [comment]
Hi!

Could you provide your config file or training log?

**Comment by STARK1234** [comment]
config and log is as follows.

[train_log.txt](https://github.com/damo-cv/TransReID/files/7150976/train_log.txt)

**Comment by michuanhaohao** [comment]
It is really puzzling. The config is no problem, but the loss is a litter great than our logs. Could you reproduce the performance on Market and Duke? If you can reproduce the performance on Market and Duke, maybe you can re-set your MSMT dataset.

**Comment by STARK1234** [comment]
Thanks a lot! I reproduced the performance on One V100.
But on 1080Ti, 1.8.1 torch only got 45% mAP and 1.6.0 torch got 58% mAP.

**Comment by michuanhaohao** [comment]
I am also confused. This is also the first time I heard about this problem. The problem may be from the fp16 training of different GPUs, pytorch versions and cuda versions.

**Comment by STARK1234** [comment]
Very appreciate for your answers!

**Comment by oliver8459** [comment]
Hi~ I have the same question about performance. I trained MSMT17_vit_transreid_stride_384 on 2 Tesla P100 gpus, batchsize=32 per gpu. So it's same to your batchsize=64 setting. But the mAP  and rank-1 decrease 1.2% and 0.5% respectively. Is it the same reason?
[train_log.txt](https://github.com/damo-cv/TransReID/files/7496055/train_log.txt)

**Comment by oliver8459** [comment]
And I use 8gpus(bs=256), mAP=65.4%(-4%), rank-1=83.8%(-2.4%) only.
